modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BetaButton.tsx

```diff
@@ -30,11 +32,12 @@ export function BetaButton() {
 					alignPosition="top"
 					disableScroll
 					header={Liferay.Language.get('beta-feature')}
-					show={true}
+					show={showPopover}
 					trigger={
 						<ClayButton
 							className="rounded-circle"
 							displayType="beta"
+							onClick={() => setShowPopover(!showPopover)}
```

Comment:
@kildarep, as we are going to remove the Tooltip we can show the Popover on the hover of the button, for that we can remove the `onClick` and add two attributes called `onMouseOver` and `onMouseOut`:

```tsx
onMouseOver={() => setShowPopover(true)}
onMouseOut={() => setShowPopover(false)}
```